### PR TITLE
XW-2780 | featured videos configured by wiki factory

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-2780

Don't merge before we'll have featuredVideos variable ready on production.

@Wikia/x-wing

### CODE REVIEW

**Review by rogatty — APPROVED** [review]
The code looks good but I'm not sure about the variable name. We named the extension `ArticleVideo` and variable is `featuredVideos`, the connection is not obvious.

File: extensions/wikia/ArticleVideo/ArticleVideo.setup.php [hunk 1]
```diff
@@ -6,16 +6,3 @@
 $wgHooks['BeforePageDisplay'][] = 'ArticleVideoHooks::onBeforePageDisplay';
 $wgHooks['MakeGlobalVariablesScript'][] = 'ArticleVideoHooks::onMakeGlobalVariablesScript';
 $wgHooks['SkinAfterBottomScripts'][] = 'ArticleVideoHooks::onSkinAfterBottomScripts';
-
-
-// cityId => articleId => videoDetails
-$wgVideoMVPArticles = [
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/ArticleVideo.setup.php:12]: Did we move this data to WikiFactory already?

Comment (latata) [reply extensions/wikia/ArticleVideo/ArticleVideo.setup.php:12]: Not yet, that's why I set Don't merge label here

File: extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div id="article-video" class="article-video">
-	<img class="video-thumbnail" src="<?= $thumbnailUrl; ?>">
+	<img class="video-thumbnail" src="<?= htmlspecialchars( $thumbnailUrl ); ?>">
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php]: `Sanitizer::encodeAttribute()`

**Review by kvas-damian — APPROVED** [review]

**Review by rogatty — APPROVED** [review]

**Pull request merged**